### DISCUSSION TITLE
Shared Albums?

### DISCUSSION THREAD

**Discussion by NineBallo** [steilerDev/icloud-photos-sync#422] [Q&A]
Are shared albums supported? I cant seem to get them to sync. It's definitly working because everything else is syncing just not my shared albums.

**Comment by steilerDev** [comment]
This is unfortunately [a known limitation of the iCloud Web API](https://github.com/steilerDev/icloud-photos-sync#limitations) (which is used by this tool). When you navigate to a folders on iCloud.com, and select "Shared Library" you get a message along the lines of "Albums in the shared library view are not yet supported on iCloud.com".

The assets in your shared library are still synced - it's only their album location that is unknown.
